courses/free-courses-en.md

```diff
@@ -80,7 +80,7 @@
 * [Web Development](#web-development)
 
 
-### 0 - MOOC
+<h3 id="0---mooc">0 - MOOC</h3>
```

Comment:
Should it better use markdown syntax and modify index item? Like in other listings. See C# in `langs` :wink:

There are currently a parser/app processing markdown: See #6643